website/docs/r/dataproc_job.html.markdown

```diff
@@ -0,0 +1,364 @@
+            "WORD_COUNTS = FOREACH GROUPS GENERATE group, COUNT(WORDS)",
+            "DUMP WORD_COUNTS"
+        ]
+    }
+}
+```
+
+* `query_list`- (Optional) The list of Hive queries or statements to execute as part of the job.
+   Conflicts with `query_file_uri`
+
+* `query_file_uri` - (Optional) HCFS URI of file containing Hive script to execute as the job.
+   Conflicts with `query_list`
+
+* `continue_on_failure` - (Optional) Whether to continue executing queries if a query fails. The default value is false. Setting to true can be useful when executing independent parallel queries. Defaults to false.
+
+* `script_variables` - (Optional) Mapping of query variable names to values (equivalent to the Pig command: `name=[value]`).
+
+* `properties` - (Optional) A mapping of property names to values, used to configure Pig. Properties that conflict with values set by the Cloud Dataproc API may be overwritten. Can include properties set in `/etc/hadoop/conf/*-site.xml`, `/etc/pig/conf/pig.properties`, and classes in user code.
+
+* `jar_file_uris` - (Optional) HCFS URIs of jar files to add to the CLASSPATH of the Pig Client and Hadoop MapReduce (MR) tasks. Can contain Pig UDFs.
+
+* `logging_config.driver_log_levels`- (Optional) The per-package log levels for the driver. This may include 'root' package name to configure rootLogger. Examples: 'com.google = FATAL', 'root = INFO', 'org.apache = DEBUG'
+
+
+The `sparksql_config` block supports:
+
+```hcl
+
+# Submit a spark SQL job to the cluster
+resource "google_dataproc_job" "sparksql" {
+    ...
+
+    sparksql_config {
+        query_list       = [
+            "DROP TABLE IF EXISTS dprocjob_test",
+            "CREATE TABLE dprocjob_test(bar int)",
+            "SELECT * FROM dprocjob_test WHERE bar > 2",
+        ]
+    }
+}
+```
+
+* `query_list`- (Optional) The list of SQL queries or statements to execute as part of the job.
+   Conflicts with `query_file_uri`
+
+* `query_file_uri` - (Optional) The HCFS URI of the script that contains SQL queries.
+   Conflicts with `query_list`
+
+* `script_variables` - (Optional) Mapping of query variable names to values (equivalent to the Spark SQL command: `SET name="value";`).
+
+* `properties` - (Optional) A mapping of property names to values, used to configure Spark SQL's SparkConf. Properties that conflict with values set by the Cloud Dataproc API may be overwritten.
+
+* `jar_file_uris` - (Optional) HCFS URIs of jar files to be added to the Spark CLASSPATH.
+
+* `logging_config.driver_log_levels`- (Optional) The per-package log levels for the driver. This may include 'root' package name to configure rootLogger. Examples: 'com.google = FATAL', 'root = INFO', 'org.apache = DEBUG'
+
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are
+exported:
+
+* `reference.cluster_uuid` - A cluster UUID generated by the Cloud Dataproc service when the job is submitted.
```

Comment:
Because of how nested objects work, this is actually `reference.0.cluster_uuid` (likewise for `status`)